### ISSUE TITLE
Have a "team eligibility" or "team makeup" status display

### ISSUE THREAD

**Issue by stenington** [Iridescent-CM/technovation-app#1386]
Throughout the whole submission period we should show teams who's officially on the team, who's pending, and what they need to do. Then we can later choose whether to drop people, prevent submission, whatever.

QA

- Login as a student on a team
  - During this QA, you will probably sign up / login as a handful of students
  - Ensure that your main login has done all of their registration steps

- Ensure that the team has 1 or more students who have not finished various registration steps
  - You may have to create new accounts and a new team to set this data up correctly
  - Try varying the registration states, like one student hasn't done anything, one student has parental consent but not their location, and one student has their location but not their parental consent

- Start your team's submission
  - Only a fully onboarded student can do this

- Verify that your onboarding students appear in the right hand sidebar when you start your submission

- Verify that you can go to "Fix this now" to get an overview of what your teammates need to do

- Verify that you can remove anyone who is still pending, because maybe that person left the team and never removed themselves

- Verify that once all the students on the team are completely onboarded, the team no longer gets a warning about not being qualified to compete

**Comment by kmorton2017** [comment]
I did the steps outlined. I confirm that I saw the on-boarding students in the righthand sidebar. 

I confirm that I went to the Fix This Now page. I removed one team member.

Once all students were completely on-boarded, I no longer got a warning about not being qualified to compete.

Question-technically the team is still eligible to compete even if all the team members are not onboarded, right? Just those particular students are ineligible to compete. They would not be considered part of the team.
